Project02Eclipse/src/com/hemebiotech/analytics/Symptom.java

```diff
@@ -0,0 +1,53 @@
+package com.hemebiotech.analytics;
+
+/**
+ * Symptom objects contain details of a specific symptom and the number of cases reported. 
+ *
+ * @version 21/9/2021
+ */
+
+public class Symptom {
+	
+	private String name;
+	private int cases;
+	
+	/**
+	* Constructor for Symptom class.
+	* Creates a new Symptom and sets reported cases to 1
+	* 
+	* @param symptom the name of the symptom
+	*/
+	public Symptom (String symptom) {
+		name = symptom;
+		cases = 1;
+	}
+	
+	/**
+	* Add a new case of this symptom
+	* 
+	* @return <code>true</code> when a new case is successfully added 
+	*/
+	public boolean addCase() {

```

Comment:
this method can return void, on the usages, you don't read the re4turning boolean, SymptomList.java #76 and #92
```
public void addCase() {
      cases++;
}
```

Reply:
true there is no need to return a boolean here, there is no alternative outcome. have updated this, thanks.